src/pages/VenuePage/VenuePage.tsx

```diff
@@ -113,6 +115,10 @@ const VenuePage: React.FC = () => {
     setLocationData({ userId, locationName: venueName });
   }, [userId, venueName]);
 
+  useEffect(() => {
+    document.title = PLAYA_VENUE_NAME + ": " + venueName;
+  }, [venueName]);
```

Comment:
As we already include `react-use` in our dependencies, we could use `useTitle` here:

- https://github.com/streamich/react-use/blob/master/docs/useTitle.md

Reply:
use that now